dvc/ui/__init__.py

```diff
@@ -1,26 +1,26 @@
 import sys
-from collections import defaultdict
 from typing import (
     TYPE_CHECKING,
     Any,
     Dict,
     Iterable,
     Optional,
     Sequence,
+    TextIO,
     Union,
 )
 
 from funcy import cached_property
 
-from dvc.progress import Tqdm
-from dvc.utils import colorize
```

Comment:
As this is supposed to be imported in `dvc.commands` in top-level, trying to make the imports lazy.